familyplanningmodule/src/main/res/drawable/rounded_corner_edit_text.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layer-list xmlns:android="http://schemas.android.com/apk/res/android">
+    <item>
+        <shape android:shape="rectangle">
+            <solid android:color="#1E81BD" />
```

Comment:
Can you move these text-colors and text-color-hints to the `colors.xml` files and reuse them